core/src/test/kotlin/com/kin/ecosystem/core/accountmanager/AccountManagerImplTest.kt

```diff
@@ -123,7 +119,7 @@ class AccountManagerImplTest : BaseTestClass() {
             firstValue.onEvent(null)
         }
 
-        assertEquals(listOf(1, 1, 2, 3, 4), stateList)
+        assertEquals(listOf(1, 1, 2, 3), stateList)
```

Comment:
once adding back REQUIRE_TRUSTLINE = 0x00000003 probably will need to also revert this line